tests/kolainst/nondestructive/misc.sh

```diff
@@ -6,13 +6,16 @@ cd $(mktemp -d)
 
 # Ensure multicall is correctly set up and working.
 R_O_DIGEST=$(sha512sum $(which rpm-ostree) | cut -d' ' -f1)
-O_C_DIGEST=$(sha512sum $(which ostree-container) | cut -d' ' -f1)
+O_C_DIGEST=$(sha512sum $(which /usr/libexec/libostree/ext/ostree-container) | cut -d' ' -f1)
```

Comment:
Do we need the which if we have the full path?

Reply:
You're right we don't!  But I don't want to respin CI for that, will fix in a followup.